File: aricode/sparring/challenges/15_arraysum/arraysum.c
/* C - Challenge 15: Array Sum, return sum%256 = 158 */
int main(void) {
    int data[100];
    for (int i = 0; i < 100; i++) data[i] = i * i;
    int sum = 0;
    for (int i = 0; i < 100; i++) sum += data[i];
    return sum % 256;
}

File: aricode/sparring/challenges/15_arraysum/arraysum.go
// Go - Challenge 15: Array Sum, return sum%256=103
package main
import "os"
func main() {
	data := make([]int, 100)
	for i := 0; i < 100; i++ { data[i] = i * i }
	sum := 0
	for _, v := range data { sum += v }
	os.Exit(sum % 256)
}
